src/perl/vdotest/VDOTest/VDOFIOPerfTest.pm

```diff
@@ -73,9 +73,7 @@ sub testFioBenchmark {
   my $rate  = $wrate + $rrate;
   my $io    = $wio   + $rio;
 
-  # Print out the SSD wearout indicators
-  # XXX Assert that we're not too worn out?  (worn out SSDs == invalid test?)
-  $machine->printSSDWearout();
+  # To do: Print out the SSD wearout indicators
 
```

Comment:
Given that we don't even have physical hardware any more, can we have meaningful SSD wearout indicators? I'd probably cut the comment, too.

Reply:
I was thinking of data collection for analysis for performance characteristics when testing on real hardware. But in our main lab testing environment with only virtual systems it's not meaningful...